File: SchoolSync_SpringBoot/backend/src/main/java/com/schoolsync/dto/FeeDTO.java
package com.schoolsync.dto;

import lombok.Data;
import java.time.LocalDate;

@Data
public class FeeDTO {
    private Long feeId;
    private Long studentId;
    private String studentName;
    private String feeType;
    private Double amount;
    private LocalDate dueDate;
    private LocalDate paidDate;
    private String status;
    private String paymentMethod;
    private String transactionId;
    private String receiptNumber;
    private String remarks;
}

File: SchoolSync in MS.NET/Backend in .NET/DTOs/FeeDto.cs
namespace SmartSchoolAPI.DTOs;

public class FeeDto
{
    public int? FeeId { get; set; }
    public int StudentId { get; set; }
    public string FeeType { get; set; } = string.Empty;
    public decimal Amount { get; set; }
    public DateTime DueDate { get; set; }
    public DateTime? PaidDate { get; set; }
    public string Status { get; set; } = string.Empty;
    public string? PaymentMethod { get; set; }
    public string? TransactionId { get; set; }
    public string? Remarks { get; set; }
}

